third_party/terraform/tests/resource_app_engine_application_test.go

```diff
@@ -57,7 +57,7 @@ resource "google_app_engine_application" "acceptance" {
   auth_domain     = "hashicorptest.com"
   location_id     = "us-central"
   serving_status  = "SERVING"
-}`, pid, pid, org)
+}`, pid, pname, org)
```

Comment:
It's not immediately apparently that this is being pulled in from the globalish variable in resource_google_project_test.go . It would be nice if it was named something to indicate that. It would be nice if go followed the C++ convention of all caps, but since it doesn't, maybe just a more explicit name. Hopefully this will prevent accidentally shadowing the variable in the future.